Renamed Variables to Placeholders (#3650)
BetonQuest/BetonQuest: CHANGELOG.md

- `menu` io based on packetevents renamed to `packetevents`
- config option `conversation.interceptor.delay` to configure a delay between the end of a conversation and the moment the interceptor is ended
- `hologram` `top:` line to customizable format

> - Variables name to Placeholder, this should not effect any user scripts and only affects the name of this concept

Reviewer: ```suggestion
- Variables renamed to Placeholders. This change won't harm any user scripts, it's only a conceptual change
```
